test/e2e/case14_selection_test.go

```diff
@@ -268,3 +270,77 @@ var _ = Describe("Test policy compliance with namespace selection", Ordered, fun
 		}
 	})
 })
+
+var _ = Describe("Test NS selection events after controller restart", Label("running-in-cluster"), Ordered, func() {
+	const (
+		configmapName = "case14-cm"
+		policyYAML    = "../resources/case14_namespaces/case14_configmaps_policy.yaml"
+		policyName    = "case14-configmaps"
+	)
+
+	BeforeAll(func() {
+		By("Create initial Namespaces")
+		namespaces := clientManaged.CoreV1().Namespaces()
+		for i := range 2 {
+			ns := "case14-" + strconv.Itoa(i)
+			if _, err := namespaces.Get(context.TODO(), ns, metav1.GetOptions{}); err != nil && errors.IsNotFound(err) {
+				Expect(namespaces.Create(context.TODO(), &corev1.Namespace{
```

Comment:
This should use the test context.

```suggestion
	BeforeAll(func(ctx SpecContext) {
		By("Create initial Namespaces")
		namespaces := clientManaged.CoreV1().Namespaces()
		for i := range 2 {
			ns := "case14-" + strconv.Itoa(i)
			if _, err := namespaces.Get(ctx, ns, metav1.GetOptions{}); err != nil && errors.IsNotFound(err) {
				Expect(namespaces.Create(ctx, &corev1.Namespace{
```